packages/cli/generators/relation/templates/controller-relation-template-belongs-to.ts.ejs

```diff
@@ -23,7 +24,7 @@ export class <%= controllerClassName %> {
         description: '<%= targetModelClassName %> belonging to <%= sourceModelClassName %>',
         content: {
           'application/json': {
-            schema: { type: 'array', items: { 'x-ts-type': <%= targetModelClassName %> } },
+            schema: { type: 'array', items: getModelSchemaRef(<%= targetModelClassName %>) },
```

Comment:
While we are touching this file & line, should we fix the white-space formatting too?

```suggestion
            schema: {type: 'array', items: getModelSchemaRef(<%= targetModelClassName %>)},
```